### PR TITLE
MFT: introduce the standalone tracking

### PR DESCRIPTION
**Pull request by bovulpes**
Write the skeleton of the first files for the tracking (as ITS).
For the moment there is only the loading of the clusters in the MFT layers (two times five disks = 10 layers).

### CODE REVIEW

File: Detectors/ITSMFT/MFT/macros/run_trac.C [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+#include "FairParRootFileIo.h"
+#include "FairSystemInfo.h"
+#include "Field/MagneticField.h"
+
+#include "MFTReconstruction/TrackerTask.h"
+
+#endif
+
+void run_trac(Int_t nEvents = 10, Int_t nMuons = 100, TString mcEngine = "TGeant3", Float_t rate = 0.)
+{
```
Comment (shahor02) [inline Detectors/ITSMFT/MFT/macros/run_trac.C]: Could you please consider interface which takes explicitly input file names as arguments, this legacy interface is not compatible with o2sim output. The ITS macros are also gradually converted to explicit names format (still keeping old interface for bwd compatibility): https://github.com/AliceO2Group/AliceO2/blob/dev/macro/run_trac_its.C

File: Detectors/ITSMFT/MFT/reconstruction/include/MFTReconstruction/Track.h [hunk 2]
```diff
@@ -9,36 +9,47 @@
 // or submit itself to any jurisdiction.
 
 /// \file Track.h
-/// \brief Simple track obtained from hits
-/// \author bogdan.vulpescu@cern.ch 
-/// \date 11/10/2016
+/// \brief Definition of the MFT track
```
Comment (shahor02) [inline Detectors/ITSMFT/MFT/reconstruction/include/MFTReconstruction/Track.h]: We move data classes to DataFormat, e.g. https://github.com/AliceO2Group/AliceO2/blob/dev/DataFormats/Detectors/ITSMFT/ITS/include/DataFormatsITS/TrackITS.h
Could you please do the same for MFT track?

File: Detectors/ITSMFT/MFT/reconstruction/include/MFTReconstruction/Track.h [hunk 3]
```diff
@@ -9,36 +9,47 @@
...
 
-  Track& operator=(const Track& track);
+    // These functions must be provided
+    //Bool_t propagate(Float_t alpha, Float_t x, Float_t bz);
+    //Bool_t update(const Cluster& c, Float_t chi2, Int_t idx);
 
-  ClassDefOverride(Track,1);
+  private:
 
+    ClassDef(Track, 1)
```
Comment (shahor02) [inline Detectors/ITSMFT/MFT/reconstruction/include/MFTReconstruction/Track.h]: Please use ClassDefNV for non-virtual classes

**Comment by bovulpes** [comment]
@shahor02 I will close this pull request and I will have a look at the changes you propose because I have another PR which I have to solve before with coding rules not very clear to me #879

**Comment by bovulpes** [comment]
After corrections I am reopening the PR.

**Comment by bovulpes** [comment]
Open it again in #917

**Pull request closed without merging**